app/assets/javascripts/discourse/lib/keyboard-shortcuts.js.es6

```diff
@@ -133,6 +135,23 @@ export default {
     Ember.run.later(() => $(".d-editor .quote").click(), 500);
   },
 
+  goToFirstSuggestedTopic() {
+    const $el = $(".suggested-topics a.raw-topic-link:first");
+    if ($el.length) {
+      $el.click();
+    } else {
+      const controller = this.container.lookup("controller:topic");
+      // Only the last page contains list of suggested topics.
+      const url = `/t/${controller.get("model.id")}/last.json`;
```

Comment:
Does this work when using subfolder?

Reply:
The `ajax` helper automatically applies the subfolder.